README.md

```diff
@@ -87,11 +87,14 @@ By default host and scheme are pulled from the provided conn but can be overridd
 config :jsonapi,
   host: "www.someotherhost.com",
   scheme: "https",
-  underscore_to_dash: true
+  underscore_to_dash: true,
+  remove_links: true
 ```
 
 Additionally, JSONAPI now recommends the use of dashes (`-`) in place of underscore (`_`) as a word separator. Enabling this change is easy with the `underscore_to_dash` option, which handles the conversion for you.
 
+`links` data can optionally be removed from the payload via the configuration above.
```

Comment:
This is a little over the scope of your PR but IF you are willing we should create a "Configuration" section which documents each key and its default. I can create a separate issue too.

Reply:
@jeregrine So one thing I noticed is this is already in the `Configuration` section.  Let me know what you think about the most recent commit: https://github.com/jeregrine/jsonapi/pull/82/files#diff-04c6e90faac2675aa89e2176d2eec7d8R94